plugins/modules/redhat_subscription.py

```diff
@@ -24,6 +24,11 @@
       I(server_proxy_hostname), I(server_proxy_port), I(server_proxy_user) and
       I(server_proxy_password) are no longer taken from the C(/etc/rhsm/rhsm.conf)
       config file and default to None.
+    - Since 6.5.0, credentials (i.e. I(username) and I(password), or I(activationkey), or I(token))
```

Comment:
1. Better specify what a version refers to (to avoid confusion).
2. Please avoid latin abbreviations (https://docs.ansible.com/ansible/devel/dev_guide/style_guide/index.html#avoid-using-latin-phrases).
```suggestion
    - Since community.general 6.5.0, credentials (that is, I(username) and I(password), or I(activationkey), or I(token))
```